crates/redeem/src/benchmarking.rs

```diff
@@ -111,16 +116,17 @@ fn test_request<T: crate::Config>(vault_id: &DefaultVaultId<T>) -> DefaultRedeem
     }
 }
 
-fn get_vault_id<T: crate::Config>() -> DefaultVaultId<T> {
-    VaultId::new(
-        account("Vault", 0, 0),
-        get_collateral_currency_id::<T>(),
-        get_wrapped_currency_id::<T>(),
-    )
+fn mint_vault_collateral<T: crate::Config + loans::Config>(vault_id: &DefaultVaultId<T>) {
+    if vault_id.collateral_currency().is_lend_token() {
+        mint_lend_tokens::<T>(&vault_id.account_id, vault_id.collateral_currency());
+    } else {
+        mint_collateral::<T>(&vault_id.account_id, 100_000u32.into());
```

Comment:
Actually, this is not required right?

Reply:
It's not required right now, I just wanted to make it easy to change the collateral used by the vault in the future